Pages/cart page/cart.html

```diff
@@ -65,14 +65,14 @@ <h4 class="title font-bold m">Easy Mart</h2>
         <main class="main grid-2-column flex-grow-1">
             <div class="flex flex-column flex-align-end pd-1">
                 <article class="cart-card m-2">
```

Comment:
Nice work on UI, very clean. Just one suggestion on Card, I think, you should not use the `article` tag for cards, as semantically it has a different meaning. `div` is fine for cards.

Reply:
Okay thanks for pointing this out, will change them to div.